### DISCUSSION TITLE
Is it possible to enable or disable queue operations in the Volcano Extends API?

### DISCUSSION THREAD

**Discussion by hzp01** [fabric8io/kubernetes-client#5666] [Q&A] [answered]
Is it possible to enable or disable queue operations in the Volcano Extends API? For instance, open queue, close queue, activate queue, or deactivate queue?

**Comment by rohanKanojia** [comment]
I am not a volcano expert but from what I see in [documentation](https://volcano.sh/en/docs/queue/#example) , it seems just updating `.status.state` to the desired queue state should set the queue state. Is there any additional process involved?

**Reply by hzp01** [reply]
The following code is ineffective.Could you please point out where the issue lies?
```
        String queueName = "test2";
        NamespacedVolcanoClient volcanoClient = VolcanoClient.getVolcanoClient();
        Queue queue = volcanoClient.queues().withName(queueName).get();
        queue.getStatus().setState("closed");
        Queue queue1 = volcanoClient.queues().resource(queue).serverSideApply();
        System.out.println(queue1);
```

**Reply by rohanKanojia** [reply, marked as answer]
As far as I remember, in order to update status there is a separate subresource call `/sratus`. Instead of `serverSideApply()` you should try using `updateStatus()`

**Reply by hzp01** [reply]
Your suggestion is correct. It works. Thank you very much.
